src/Providers/ApiLogServiceProvider.php

```diff
@@ -50,7 +52,16 @@ public function bindServices(){
                 $instance = DBLogger::class;
                 break;
             default:
-                throw new Exception("Unsupported Driver");
+                try {
```

Comment:
There's probably a much better way of doing this.
I tested it several times and this does work.
Any recommendations appreciated